test/functional/feature_notifications.py

```diff
@@ -84,22 +84,11 @@ def run_test(self):
         assert_equal(sorted(blocks), sorted(os.listdir(self.blocknotify_dir)))
 
         if self.is_wallet_compiled():
+            self.connect_nodes(0, 1)
             self.log.info("test -walletnotify")
             # wait at most 10 seconds for expected number of files before reading the content
             self.wait_until(lambda: len(os.listdir(self.walletnotify_dir)) == block_count, timeout=10)
 
-            # directory content should equal the generated transaction hashes
-            tx_details = list(map(lambda t: (t['txid'], t['blockheight'], t['blockhash']), self.nodes[1].listtransactions("*", block_count)))
-            self.stop_node(1)
-            self.expect_wallet_notify(tx_details)
```

Comment:
shouldn't this be adjusted to test that the rescan RPC also notifies?

Reply:
Good point, done.